floto/static/util.js

```diff
@@ -1,6 +1,6 @@
-function fetch_with_retry(url, callback, error_callback=base_error_callback, backoff_count=0, query_params=get_query_params()) {
-  url += "?" + query_params
-  fetch(url).then((response) => {
+function fetch_with_retry(url, callback, error_callback=base_error_callback, backoff_count=0) {
+  let updated_url = with_query_params(url)
```

Comment:
This used to re-append parameters each failure, so it didn't properly retry. Now it works.